src/style/aframe.css

```diff
@@ -31,6 +33,24 @@ body {
   width: 100%;
 }
 
+.a-canvas.fullscreen {
```

Comment:
using qualifiers is mainly to avoid collisions with existing class in the page. if we make fullscreen specific to a-canvas the prefix is no longer necessary and makes things a bit prettier to read and write.